Foundation/NSPropertyList.swift

```diff
@@ -117,6 +138,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return true
     } else if input === kCFBooleanFalse {
         return false
+    } else if CFGetTypeID(input) == _CFKeyedArchiverUIDGetTypeID() {
```

Comment:
Hmm that seems to be a mistake if it does not call CFGetTypeID on itself for NSCFType; I need to verify that bridging works correctly with that change - I think that can probably be improved outside of this pull request (and a follow up commit addresses this issue)